demo/src/docs/ColorPalette.js

```diff
@@ -0,0 +1,256 @@
+import React, { Fragment } from "react";
+import { Link } from "react-router-dom";
+import withStyles from "elevate-ui/withStyles";
+import classNames from "classnames";
+import isObject from "lodash/isObject";
+import Color from "color";
+import Typography from "elevate-ui/Typography";
+import Alert from "elevate-ui/Alert";
+
+const ColorPalette = ({ classes, theme }) => {
+  return (
+    <div className={classes.wrapper}>
+      <div className={classes.headingContainer}>
+        <Typography type="title" gutterBottom>{`Color Palette`}</Typography>
+        <Typography type="body" className={classes.subheading}>
+          All colors in the theme object are displayed here, with 10 shade
+          variants for each base color. This wide spectrum gives us the ability
+          to find a color for just about any possible use case.
+        </Typography>
+      </div>
+      <div className={classes.alertContainer}>
+        <Alert color="warning">
```

Comment:
We are using the alert color of warning here, when its more of a Tip. I noticed on the Padding view we use an alert with the blue color. Maybe that would be more on target for the color of this info box?